lib/mux-webhook-processor.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  sendSlackAssetReady,
+  sendSlackAutoDeleteMessage,
+} from './slack-notifier';
+import { getScores as moderationGoogle } from './moderation-google';
+import { getScores as moderationHive } from './moderation-hive';
+import { autoDelete } from './moderation-action';
+
+type WebhookRequestBody = {
+  type: string;
+  object: any;
+  id: string;
+  environment: any;
+  data: any;
+  playback_ids?: any[];
+  duration: number;
+  created_at: string;
+  accessor_source: string;
+  accessor: string;
+  request_id: string;
+}
```

Comment:
Is there a place where I can get all available fields of the webhook request?

Reply:
There has been some work on this recently by techops team, ask in #devex-dev channel

Reply:
Look at the `spec/webhook/samples` directory in this PR: https://github.com/muxinc/openapi-specification/pull/181

Reply:
Thank you @jaredsmith :)